From 5cc81108739c8b5eb5679f726ae86e677e9dee65 Mon Sep 17 00:00:00 2001
From: Brad Bishop <bradleyb@fuzziesquirrel.com>
Date: Thu, 5 Sep 2019 20:39:57 -0400
Subject: [PATCH] arm: evb-ast2600: put environment back at 512KiB

Prior to 473f430b90 the environment was at 512KiB, and compatible with
OpenBMC.  Restore OpenBMC compatibility by moving the environment back
to 512KiB.

Upstream-Status: Pending
Upstream-Status: https://github.com/AspeedTech-BMC/u-boot/pull/1
Signed-off-by: Brad Bishop <bradleyb@fuzziesquirrel.com>
---
 include/configs/evb_ast2600.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/evb_ast2600.h b/include/configs/evb_ast2600.h
index b4de42a05a..3a12f2f0d4 100644
--- a/include/configs/evb_ast2600.h
+++ b/include/configs/evb_ast2600.h
@@ -18,7 +18,7 @@
 
 /* Environment */
 #define CONFIG_ENV_SIZE			0x10000
-#define CONFIG_ENV_OFFSET		0x90000
+#define CONFIG_ENV_OFFSET		0x60000
 #define CONFIG_ENV_SECT_SIZE		(4 << 10)
 
 #endif	/* __CONFIG_H */
-- 
2.21.0

